src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -3273,6 +3273,7 @@ internal static bool IsSynthesizedAttributeOptional(WellKnownMember attributeMem
                 case WellKnownMember.System_STAThreadAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_IteratorStateMachineAttribute__ctor:
+                case WellKnownMember.System_Runtime_CompilerServices_TupleElementNamesAttribute__ctorTransformNames:
                     return true;
```

Comment:
Possibly. I will check.

---
In reply to: [79673650](https://github.com/dotnet/roslyn/pull/13923#discussion_r79673650) [](ancestors = 79673650)